paddle/benchmark_convnet_model.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import functools
+import numpy as np
+import time
+
+import cProfile, pstats, StringIO
+
+import paddle.v2 as paddle
+import paddle.v2.fluid as fluid
+import paddle.v2.fluid.profiler as profiler
+
+
+def parse_args():
+    parser = argparse.ArgumentParser('Convolution model benchmark.')
+    parser.add_argument(
+        '--model',
+        type=str,
+        default='resnet',
+        help='The model architecture: (resnet).')
+    parser.add_argument(
+        '--batch_size', type=int, default=32, help='The minibatch size.')
+    parser.add_argument(
+        '--iterations', type=int, default=100000, help='The number of minibatches.')
+    parser.add_argument(
+        '--pass_num', type=int, default=100, help='The number of minibatches.')
+    parser.add_argument(
+        '--order',
+        type=str,
+        default='NCHW',
+        help='The data order: (NCHW, NHWC).')
+    parser.add_argument(
+        '--device', type=str, default='GPU', help='The data order: (GPU, CPU).')
```

Comment:
You can use "[choices](https://docs.python.org/3/library/argparse.html#choices)" in parser.add_argument.

Reply:
Done.